recipe/run_test.sh

```diff
@@ -24,9 +24,13 @@ ncks -O --rgr grid=grd_2x2.nc \
 
 
 ncap2 -O -s 'tst[lat,lon]=1.0f' skl_t42.nc dat_t42.nc
-if [[ $(uname) == Darwin ]]; then
-  ncremap -a conserve -s grd_t42.nc -g grd_2x2.nc -m map_t42_to_2x2.nc  # FIXME: This call for reason prevent CircleCI from uploading the package!
-  ncremap -i dat_t42.nc -m map_t42_to_2x2.nc -o dat_2x2.nc
-  ncwa -O dat_2x2.nc dat_avg.nc
-  ncks -C -H -v tst dat_avg.nc
-fi
+
+# This test is hanging on OS X and we don't execute it on Linux due to an odd
+# issue with CircleCI not uploading the binary after the test is executed.
```

Comment:
This was fixed on our CircleCI setup some time ago

Reply:
Good to know. I'll re-try it. Thanks!